test/dlc_tests/sanity/test_dlc_labels.py

```diff
@@ -42,4 +44,71 @@ def test_dlc_major_version_label(image, region):
 
     assert major_version, f"{image} has no LABEL named 'dlc_major_version'. Please insert label."
 
-    LOGGER.info(f"{image} has 'dlc_major_version' = {major_version}")
+    test_utils.LOGGER.info(f"{image} has 'dlc_major_version' = {major_version}")
+
+
+@pytest.mark.integration("dlc_major_version_label")
+@pytest.mark.model("N/A")
+def test_dlc_version_dockerfiles(image):
```

Comment:
Could it be better to change the name to "test_dlc_major_version_dockerfiles"?

Reply:
sure, can update the name